fbbt/src/ontology/fbbt-edit.obo

```diff
@@ -73033,22 +73033,25 @@ creation_date: 2008-05-15T05:24:58Z
 
 [Term]
 id: FBbt:00067000
-name: antennal trichoid sensillum at3
-def: "Olfactory trichoid sensillum of antennal segment 3 that is innervated by 3 ORNs. It is distinguishable from at4 by its odor response profile (de Bruyne and Baker, 2008) and the identity of the ORNs that innervate it (2a, 19a/b and 43a; Couto et al., 2005; de Bruyne and Baker, 2008)." [FlyBase:FBrf0187305, PMID:18535862]
+name: antennal intermediate sensillum ai3
+def: "Olfactory intermediate sensillum of antennal segment 3 that is innervated by 3 ORNs (2a, 19a/b and 43a)." [FBC:CP, FlyBase:FBrf0187305, FlyBase:FBrf0224383, FlyBase:FBrf0229748, PMID:18535862]
+comment: Reclassified from trichoid to intermediate by fluorescence-guided single sensillum recording (FG-SSR) (Lin and Potter, 2015; FBC:CP).
 subset: olfactory_system_draft
+synonym: "antennal trichoid sensillum at3" NARROW []
```

Comment:
Perhaps make this an 'exact' synonym. Both names do refer to the same thing.

Reply:
Or perhaps I'm not following the changes here.  I see the same synonym bellow as exact on at4...

Reply:
Yeah, this paper (FBrf0224383) decided to rename at4 to at3, then two of the same authors subsequently decided that this was a bad idea in FBrf0229748.